omnipaxos_core/tests/config/test.conf

```diff
@@ -34,7 +33,6 @@ recovery_test {
     ble_hb_delay: 3
     increment_delay: 2
     num_proposals: 20
-    storage_type: persistent
```

Comment:
is this not needed anymore? I thought it would be used here:

https://github.com/haraldng/omnipaxos/blob/cfb37839a9e1abd04664ded86c3476f0b524e021/omnipaxos_core/tests/consensus_test.rs#L27